CONTRIBUTING.md

```diff
@@ -38,23 +67,16 @@ We are currently managing payment via Upwork. If you'd like to be paid for your
     [gpg]
         program = gpg
     ```
-1. [Open a PR](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork). Be sure to fill in all the required information on the PR template.
-1. An Expensify engineer will be automatically assigned to review your PR
-1. You will need all checks to pass:
-	1. CLA - You must sign our [Contributor License Agreement](https://github.com/Expensify/Expensify.cash/blob/master/CLA.md) by following the CLA bot instructions that will be posted on your PR
-	1. Tests - All tests must pass before a merge of a pull request
-	1. Lint - All code must pass lint checks before a merge of a pull request
-
-#### Testing
-Upon submission of a PR, please include a numbered list of explicit testing steps for each platform (Web, Desktop, iOS, and Android) to confirm the fix works as expected and there are no regressions.
-
-#### Review Tips
+9. Open a pull request, and make sure to fill in the required fields.
```

Comment:
I think we should re-add the GH link for creating a pull request from a fork here so it's clear.

Reply:
That's a candid mistake on my end, adding it now and resolving this one.